sip-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -31,23 +24,45 @@
             <artifactId>lombok</artifactId>
         </dependency>
 #end
+        <dependency>
+            <groupId>de.ikor.sip.foundation</groupId>
+            <artifactId>sip-integration-starter</artifactId>
+        </dependency>
+        <!-- test dependencies -->
+        <dependency>
```

Comment:
If we want this dependency on the each adapter (which seams like a reasonable choice), we should add the dependency to sip-integration-starter.